### DISCUSSION TITLE
Is it possible to create an item that executes a command?

### DISCUSSION THREAD

**Discussion by slayerzim** [aieuo/Mineflow#217] [Q&A]
I would like to know if there is a way to create a system that when the player uses an item or consumes it, he executes a command, if possible, please help me.

**Comment by ed-medz** [comment]
Here is a recipe that may be useful to you.
![Screenshot_20240107_195711](https://github.com/aieuo/Mineflow/assets/112836351/f97bab86-c132-491a-9b5a-21acc97e7474)
